kinetic/distribution.yaml

```diff
@@ -6216,6 +6216,16 @@ repositories:
       url: https://github.com/ros-drivers/kvh_drivers.git
       version: master
     status: maintained
+  kvh_geo_fog_3d:
+    doc:
+      type: git
+      url: https://github.com/MITRE/kvh_geo_fog_3d.git
+      version: master
```

Comment:
Just checking, the repo has a `kinetic-devel` branch. Is `master` correct, or should this be `kinetic-devel`?

Reply:
I envisioned that the documentation will live in master, while kinetic-devel, melodic-devel, etc would track code releases. Is this a generally acceptable way of tracking ROS repos? The documentation will remain consistent across ROS distros.

Reply:
Sounds find to me. You may want the `source` branch to point to `kinetic-devel` though, since that's what users will get if they do `rosinstall_generator ... --upstream-devel`

Reply:
Indeed. Corrected.